combat-trainer.lic

```diff
@@ -721,6 +772,24 @@ class SpellProcess
     prepare_spell(data, game_state)
   end
 
+  def interrupt_casting(game_state)
+    fput('release spell') if game_state.casting
+    game_state.casting = false
+    game_state.cast_timer = nil
+  end
+
+  def check_consume(game_state)
+    return unless @is_necro
+    return if !game_state.prepare_consume
+    if game_state.casting && !game_state.casting_consume
```

Comment:
Yes it should be. the line before is checking if the prepare_consume flag is false and the conjunction is checking if we are casting a spell but that spell is not CF. I should use unless in the first statement, the `if !` is just a habit from using other languages